benchmarks/01_trivial/07_loop_unroll_varying-cpu.ispc

```diff
@@ -0,0 +1,412 @@
+#include "../../ispcrt/ispcrt.isph"
+
+
+// Macros
+
+#define UNROLL_VARYING_SIG_CPU(T_FOR, T_ISPC, UNROLL_FACTOR)              \
+    export void unroll_varying_cpu_##T_FOR##_##T_ISPC##_##UNROLL_FACTOR(  \
+        uniform T_ISPC src[],                                             \
+        uniform T_ISPC dst[],                                             \
+        uniform uint64 num_elems                                          \
+    )
+
+#define UNROLL_VARYING_FOREACH_BODY   \
+    foreach (i = 1 ... num_elems) {   \
+        dst[i] = src[i] + src[i - 1]; \
+    }
+
+#define UNROLL_VARYING_FOR_BODY                                               \
+    for (uint64 i = programIndex + 1ull; i < num_elems; i += programCount) {  \
+        dst[i] = src[i] + src[i - 1ull];                                      \
+    }
+
+
+
+// uint64
+
+UNROLL_VARYING_SIG_CPU(foreach, uint64, 1) {
+    dst[0] = src[0];
+    #pragma nounroll
+    UNROLL_VARYING_FOREACH_BODY
```

Comment:
NIT: having `;` after a macro use is good style. It happened that current definition of a macro is a loop body, which doesn't need a `;` after it. But having `;` safeguards for any macro definition and look more native in C/C++ program.